src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala

```diff
@@ -231,7 +231,7 @@ object AixPasswordHashAlgo extends Loggable {
       // was introduced in Java 8.
       // Fallback to ssha1 hash.
       logger.error("Your current Java installation does not support PBKDF2WithHmacSHA512 algorithm, " +
-          "which is necessary for {ssha256} hash. Falling back to {ssha1} hashing scheme")
+          "which is necessary for {ssha512} hash. Falling back to {ssha1} hashing scheme")
```

Comment:
Looks good to me for the wording, but shouldn't we fall back to ssha256 first? (which would of course fall back to ssha1 in turn if not available)